### ISSUE TITLE
audio driver error??

### ISSUE THREAD

**Issue by mahalex23707** [hydrogen-music/hydrogen#413]
Upon start up i am met with a message that says something like error audio driver, and now i am unable to play anything out of the program, i literally just downloaded the program last night and made a track or two. i uninstalled and re-installed but still the same message.

**Comment by mauser** [comment]
Hi! 

Please let us know what the message says exactly. Which version version of hydrogen are you using on which operating system?

**Comment by mahalex23707** [comment]
i downloaded the 0.97 beta version

**Comment by mahalex23707** [comment]
i am using the 0.9.7 beta version

**Comment by mikotoiii** [comment]
Just going to take a stab in the dark here. Under tools, preferences, in
the audio system tab, you can change the audio driver from auto to
PortAudio. I've seen auto not work on Windows a couple times on earlier
builds.

**Comment by mahalex23707** [comment]
Ok well I discovered that there was an issue with my audio driver because
sound refused to play from any program, and all with similar error
notifications. There was no error in troubleshooting, not was there error
in updates. I just went to advanced options and restored default settings
and restarted the computer and now it all works again. Not sure if the
initial downloading of hydrogen caused this bug but not it is resolved.
